docs/storage/prometheus.md

```diff
@@ -81,7 +81,7 @@ Metric name | Type | Description | Unit (where applicable) | -disable_metrics pa
 `container_perf_events_total` | Counter | Scaled counter of perf core event (event can be identified by `event` label and `cpu` indicates the core for which event was measured). See [perf event configuration](../runtime_options.md#perf-events). | | | libpfm
 `container_perf_metric_scaling_ratio` | Gauge | Scaling ratio for perf event counter (event can be identified by `event` label and `cpu` indicates the core for which event was measured). See [perf event configuration](../runtime_options.md#perf-events). | | | libpfm
 `container_processes` | Gauge | Number of processes running inside the container | | process |
-`container_referenced_bytes` | Gauge |  Container referenced bytes during last measurements cycle based on Referenced field in /proc/smaps file, with /proc/PIDs/clear_refs set to 1 after defined number of cycles configured through `referenced_reset_interval` cAdvisor parameter.</br>Warning: this is intrusive collection because can influence kernel page reclaim policy and add latency. Refer to https://github.com/brendangregg/wss#wsspl-referenced-page-flag for more details. | bytes | referenced_memory |
+`container_referenced_bytes` | Gauge |  Container referenced bytes during last measurements cycle based on Referenced field in /proc/smaps file(or /proc/smaps_rollup). Read frequency is controlled through `referenced_read_interval` command line parameter. Referenced bytes counter is restarted with frequency set by `referenced_reset_interval` interval by setting /proc/PIDs/clear_refs to 1.</br>Warning: this is intrusive collection because can influence kernel page reclaim policy and add latency. Refer to https://github.com/brendangregg/wss#wsspl-referenced-page-flag for more details. | bytes | referenced_memory |
```

Comment:
In my opinion it should be mentioned, that read and reset frequency may vary from 1 to n. It may be important, when somebody passes big value.

Reply:
Frequency is constant. Only the start time may vary. Offset in relation to other process differs

Reply:
Why have you decided that start time may vary?

Reply:
Say, for the sake of an argument, that there is 50 containers on a machine, each with 10 pids in it. Cadvisor starts and starts to read the smaps. As container object creation is almost instant and read is parallel, we can assume that at given point in time we are reading 500 smaps files in parallel. And as smaps reading, thanks  to how it works, is cpu intensive, every `referenced_read_interval` we get a spike in cpu utilization. Therefore to avoid this spike, later in code I enforce a random offset before the reading loop starts. The frequency is the same, but the phase is different.

Reply:
This should be documented with explanation why this is needed.

Reply:
I will put it into runtime options